.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -2,32 +2,35 @@
 name: Bug report
 about: Submit a bug report
 title: ''
-labels: t:Bug report
+labels: needs triage
 assignees: ''
 
 ---
 
-**Describe the bug**
+**Description**  
 A clear and concise description of what the bug is, and why you consider it to be a bug.
 
-**Area for Triage**: 
+**Area for Triage**:  
 <!-- See https://github.com/actions/virtual-environments/tree/main/triage-rules.yml for areas -->
 
-**Question, Bug, or Feature?**: 
+**Question, Bug, or Feature?**:  
 <!-- Choose "Question", "Bug", or "Feature" -->
 
-**Virtual environments affected**
+**Virtual environments affected**  
 - [ ] macOS 10.15
 - [ ] Ubuntu 16.04 LTS
 - [ ] Ubuntu 18.04 LTS
 - [ ] Ubuntu 20.04 LTS
 - [ ] Windows Server 2016 R2
 - [ ] Windows Server 2019
 
-**Expected behavior**
+**Expected behavior**  
 A description of what you expected to happen.
 
-**Actual behavior**
+**Actual behavior**  
+A description of what you are seeing

```

Comment:
Maybe update sentence to something like `A description of what is actually happening.` to match the "expected" section?

Reply:
Good point, fixed!